common/src/main/java/com/wynntils/wynn/model/LootrunModel.java

```diff
@@ -600,7 +607,8 @@ public List<LootrunInstance> getLootruns() {
             }
```

Comment:
Another question is: why do we not just list the files in the lootrun dir? It would have been blazingly fast. And read and parse the json only if we try to read in the lootrun. The code above in the loop looks just wrong.

Reply:
Like, if we have malformed json in a single lootrun, that will prevent us from even listing the files.

Reply:
We need to parse it to tell the user info about the lootrun. And we do not want to parse on click.

Reply:
I can change the code to not fail on a single malformed file though.

Reply:
What do we tell the user? I apparently need to test this. I just throught we listed the name of the lootrun.

Reply:
> Like, if we have malformed json in a single lootrun, that will prevent us from even listing the files.

Actually, no. We do catch errors, log it, but we can show other files. This looks good IMO.

Reply:
![image](https://user-images.githubusercontent.com/49001742/208773924-a6fdc225-6f9c-4aa4-bfa3-89c32cc9647f.png)